ipatests/setup.py

```diff
@@ -73,10 +73,14 @@
             "pytest_multihost",
             "python-ldap",
             "six",
-            "pexpect",
         ],
         extras_require={
-            "integration": ["dbus-python", "pyyaml", "ipaserver"],
+            "integration": [
+                "dbus-python",
+                "pexpect",
```

Comment:
I really was not sure where to put this dependency, can you please clarify why it should be in extras?
For me it looks same as pytest_multihost - it is required only by integration testst and those tests can not even start without it.

Reply:
The dependency should not be needed by client-only tests, only for integration tests. In theory any dependency that is only used by ``ipatests.pytest_ipa.integration`` should be in ``extras_require`` for ``integration`` (hence the name).

Something changed since I last touched the code. The PR is failing because something triggers an important of ``ipatests.pytest_ipa.integration``.

Reply:
Sounds reasonable. But should not pytest_multihost also be moved to extras?